docs/source/DeveloperDocumentation/VUDRM-token.md

```diff
@@ -122,6 +123,9 @@ The structure of the session information in a policy should be as follows:
 }
 ```
 
+### License Renewal
+To enable automatic license renewal `can_renew` must be set to true; you must also have set `polend` to be more than 60 seconds in the future or have set `duration_persist` to be more than 60. If this criteria is met then 60 seconds before the license is due to expire a request will be automatically to renew the license. Should a license renewal request fail, then it will be retried every 20 seconds until the license being renewed has expired.
```

Comment:
Could we change 'if this criteria is met' to 'if these criteria are met... etc'?
Also, the should latter half of the sentence say 'a request will be made automatically'?